code/game/objects/items/devices/healthanalyzerpro.dm

```diff
@@ -45,8 +47,28 @@
 	last_scantime = 0
 
 /obj/item/weapon/autopsy_scanner/healthanalyzerpro/AltClick()
+	if(folded)
+		to_chat(usr, "<span class='warning'>Unfold it first!</span>")
+		return
 	if(usr.is_holding_item(src))
 		toggle_mode()
+		
+/obj/item/weapon/autopsy_scanner/healthanalyzerpro/CtrlClick(mob/living/user)
+	read_log(user)
+
+/obj/item/weapon/autopsy_scanner/healthanalyzerpro/proc/fold_scanner()
+	if(usr.is_holding_item(src))
+		usr.playsound_local(usr, 'sound/misc/metal_drag3.ogg', 30, 0, 30000, TRUE, 0, FALSE)
+		if(do_after(usr, src, 3 SECONDS))
```

Comment:
3 seconds seems crazy long for this. It takes a half a second to unfold a folding phone or even a laptop while standing up. The folding feature really isn't something that can be abused in any meaningful way, so I don't see any issue with reducing this significantly.

Reply:
Testing has shown that this is an adequate amount of time to unfold what is essentially a mobile full-body scanner.